project.godot

```diff
@@ -45,36 +50,21 @@ _global_script_class_icons={
 "Game": "",
 "Interactable": "",
 "Level": "",
+"Player": "",
 "SpeechBubble": ""
 }
 
 [application]
 
 config/name="gd-wildjam18"
-run/main_scene="res://sources/menu/Splash.tscn"
 config/icon="res://icon.png"
 
-[autoload]
-
-Utils="*res://sources/Utils.gd"
-
-[input]
-
-interact={
-"deadzone": 0.5,
-"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":69,"unicode":0,"echo":false,"script":null)
- ]
-}
-
 [layer_names]
 
-2d_physics/layer_1="player"
-2d_physics/layer_2="interactable"
-2d_physics/layer_3="obstacle"
-
-[locale]
-
-translations=PoolStringArray( "res://assets/translation/text.en.translation", "res://assets/translation/text.es.translation" )
+2d_physics/layer_1="Platform"
```

Comment:
Renaming the layers already defined is probably not what you wanted to do here. Use layer after the third one if you need more (but again, do we need a layer for Wall and Platform? Obstacle isn't enough? Is this relevant to player movement?